lcadluascript/builders/customentity.cpp

```diff
@@ -21,8 +21,8 @@ const kaguya::LuaRef& CustomEntityBuilder::snapFunction() const {
     return _snapFunction;
 }
 
-bool CustomEntityBuilder::checkValues() {
-    return InsertBuilder::checkValues() && 
+bool CustomEntityBuilder::checkValues(bool throwExceptions) const{
+    return InsertBuilder::checkValues(throwExceptions) && 
            _snapFunction.type() == LUA_TFUNCTION &&
```

Comment:
Those checks won't throw exceptions if it's incorrect

Reply:
Thanks for the review! Added these missing exceptions.